apps/client/app/components/selectList.tsx

```diff
@@ -0,0 +1,25 @@
+"use client"
+import React from 'react'
+
+interface SelectProps {
+    list: KeyValuePair[];
+    selected: string;
+}
+
+export default function SelectList({list, selected = ''}: SelectProps) {
+    const handleSort = (value: string) => {
+        console.log(value)
+        window.location.href = `http://localhost:4200/purchase-orders?sort=${value}`
+    }
+
+
+    
+  return (
+    //@ts-ignore
+    <select defaultValue={selected} onChange={(e) => handleSort(e.nativeEvent.target?.value)}>
```

Comment:
ts wasn't recognizing the .value variable in the .target object. Any suggestion why? Would like to remove the @ts-ignore.